pkg/terraform/terraform.go

```diff
@@ -50,7 +50,13 @@ func (m *Mixin) getOutput(outputName string) ([]byte, error) {
 	cmd := m.NewCommand("terraform", "output", outputName)
 	cmd.Stderr = m.Err
 
+	// Terraform appears to auto-append a newline character when printing outputs
+	// Trim this character before returning the output
 	out, err := cmd.Output()
+	if len(out) > 0 && out[len(out)-1] == '\n' {
+		out = out[0 : len(out)-1]
+	}
+
```

Comment:
Are there any local terraform providers that we could run as a unit test? I'd love to get some tests behind this so that changes like this can be tested (in the future, not for this PR).

In the meantime, here is a go library function that simplifies what you have so that we aren't itching to test it as much


```suggestion
	out = bytes.TrimRight(out, "\n")

```

Reply:
Oh sweet!  Thank you for the `bytes` reference.

Will look into test coverage - thanks for the nudge!